recipes/pytango/meta.yaml

```diff
@@ -0,0 +1,67 @@
+{% set name = "pytango" %}
+{% set version = "9.3.3" %}
+{% set sha256 = "c09fc7a44cac8c2d95d3ad39ba644998e8f468c730dc495443a74de05a07e9f9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: true  # [not linux]
+
+requirements:
+  build:
+    - {{ compiler('cxx') }}
+  host:
+    - python
+    - pip
+    - cppzmq
+    - cpptango >=9.3,<9.4
```

Comment:
We might get some issues when we will be releasing some 9.4 pre-release versions because they will be considered as <9.4 
For instance, I have such pre-release test versions on my Reyn4ld conda channel (https://anaconda.org/Reyn4ld/cpptango/files).
I don't know what would be the correct upper value we should put to say < any 9.4 version, including the release candidates, alpha and beta versions.

Reply:
Good catch. It should be `<9.4.0a0` to exclude alpha, beta and rc.